lib/request-log.js

```diff
@@ -0,0 +1,59 @@
+var format = require('util').format;
```

Comment:
@thisandagain I've gotten bunyan logging working.

looks like bunyan doesn't have as much useful logging information as I'd hoped for though